extensions/wikia/AdEngine/js/context/adContext.js

```diff
@@ -190,7 +199,7 @@ define('ext.wikia.adEngine.adContext', [
 
 		for (taboolaSlot in taboolaConfig) {
 			if (taboolaConfig.hasOwnProperty(taboolaSlot) && taboolaConfig[taboolaSlot].recovery) {
-				for (i=0; i<taboolaConfig[taboolaSlot].recovery.length; i++) {
+				for (i = 0; i < taboolaConfig[taboolaSlot].recovery.length; i++) {
 					if (geo.isProperGeo(taboolaConfig[taboolaSlot].recovery[i])) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ANestedIfDepth)